Sources/HummingbirdCore/Server/HTTP/HTTP1Channel.swift

```diff
@@ -22,15 +22,48 @@ import NIOHTTPTypesHTTP1
 public struct HTTP1Channel: ServerChildChannel, HTTPChannelHandler {
     public typealias Value = NIOAsyncChannel<HTTPRequestPart, HTTPResponsePart>
 
+    /// HTTP1Channel configuration
+    public struct Configuration: Sendable {
+        /// Additional channel handlers to add to channel
+        public var additionalChannelHandlers: @Sendable () -> [any RemovableChannelHandler]
+        /// Time before closing an idle channel
+        public var idleTimeout: TimeAmount?
+
+        ///  Initialize HTTP1Channel.Configuration
+        /// - Parameters:
+        ///   - additionalChannelHandlers: Additional channel handlers to add to channel
+        ///   - idleTimeout: Time before closing an idle channel
+        public init(
+            additionalChannelHandlers: @autoclosure @escaping @Sendable () -> [any RemovableChannelHandler] = [],
```

Comment:
Likewise, this property name can now maybe reflect the _position_ of the handlers?

Reply:
Fixed comment